strapi/strapi
# fix(documentation): add check to password input

### What does it do?

Add right validations to password input on documentation plugin

### How to test it?

Go to the documentation plugin in the settings page and try to setup a new password. It should have:
1. At least one uppercase letter
2. At least one lowercase letter
3. At least one number
